package.json

```diff
@@ -8,18 +8,15 @@
   "scripts": {
     "start": "node build",
     "start:dev": "nodemon src --exec babel-node",
-    "build": "babel src -d build",
+    "build": "yarn clean:build && webpack --mode production src/",
+    "clean:build": "rimraf build",
```

Comment:
You can use `clean:build` to do the build as well by `rimraf build && yarn build`

Reply:
Done!